installing/installing_alibaba/installing-alibaba-assisted-installer.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-alibaba-assisted-installer"]
+= Installing a cluster on Alibaba Cloud using the Assisted Installer
+include::_attributes/common-attributes.adoc[]
+:context: installing-alibaba-assisted-installer
+
+toc::[]
+
+From {product-title} 4.16 and later, you can use the {ai-full} to install an {product-title} cluster on {alibaba}. {alibaba} provides a broad range of cloud computing and data storage services to online businesses and global enterprises.
```

Comment:
If there's not a specific reason to call out the version, the first clause seems optional to me.